webapp/javascript/components/Sidebar.tsx

```diff
@@ -71,6 +72,10 @@ export default function Sidebar2() {
     isRouteActive('/adhoc-single') ||
     isRouteActive('/adhoc-comparison') ||
     isRouteActive('/adhoc-comparison-diff');
+  const isSettingsActive =
+    isRouteActive('/settings') ||
+    isRouteActive('/settings/users') ||
+    isRouteActive('/settings/api-keys');
```

Comment:
Can we be lazy and use `isRouteActive(/settings*/)` that requires updating that `isRouteActive` function to support regex but it should be fine?

My worry is having to come back to update this function every time we add a new tab to the settings.